buildlib/azure-pipelines-perf.yml

```diff
@@ -31,6 +33,28 @@ stages:
           - ucx_perf
 
         steps:
+          - bash: |
+              set -xeE
+              lock_file="$(SharedDir)/on-air.lock"
```

Comment:
Aren't there any standard GHA mechanisms to prevent parallel workflow runs?
CC @B-a-S

Reply:
Nope. First thing I checked.